tests/ci/cdk/cdk/linux_docker_image_batch_build_stack.py

```diff
@@ -1,49 +1,91 @@
+
+        github_repo_owner = GITHUB_REPO_OWNER
+        github_repo_name = GITHUB_REPO_NAME
 
         # Define CodeBuild resource.
         git_hub_source = codebuild.Source.git_hub(
-            owner=GITHUB_REPO_OWNER,
-            repo=GITHUB_REPO_NAME,
+            owner=github_repo_owner,
+            repo=github_repo_name,
             webhook=False,
             branch_or_ref=GITHUB_SOURCE_VERSION,
-            clone_depth=1)
+            clone_depth=1,
+        )
 
         # Define a role.
-        code_build_batch_policy = iam.PolicyDocument.from_json(code_build_batch_policy_in_json([id]))
+        code_build_batch_policy = iam.PolicyDocument.from_json(
+            code_build_batch_policy_in_json([id], env)
+        )
         ecr_repo_names = [LINUX_AARCH_ECR_REPO, LINUX_X86_ECR_REPO]
-        ecr_power_user_policy = iam.PolicyDocument.from_json(ecr_power_user_policy_in_json(ecr_repo_names))
-        inline_policies = {"code_build_batch_policy": code_build_batch_policy,
-                           "ecr_power_user_policy": ecr_power_user_policy}
-        role = iam.Role(scope=self,
-                        id="{}-role".format(id),
-                        assumed_by=iam.ServicePrincipal("codebuild.amazonaws.com"),
-                        inline_policies=inline_policies)
+        ecr_power_user_policy = iam.PolicyDocument.from_json(
+            ecr_power_user_policy_in_json(ecr_repo_names, env)
+        )
+        inline_policies = {
+            "code_build_batch_policy": code_build_batch_policy,
+            "ecr_power_user_policy": ecr_power_user_policy,
+        }
+        role = iam.Role(
+            scope=self,
+            id="{}-role".format(id),
+            assumed_by=iam.ServicePrincipal("codebuild.amazonaws.com"),
+            inline_policies=inline_policies,
+        )
 
         # Create build spec.
-        build_spec_content = YmlLoader.load("./cdk/codebuild/linux_img_build_omnibus.yaml")
+        build_spec_content = YmlLoader.load(
+            "./cdk/codebuild/linux_img_build_omnibus.yaml"
+        )
 
         # Define environment variables.
         environment_variables = {
-            "AWS_ACCOUNT_ID": codebuild.BuildEnvironmentVariable(value=AWS_ACCOUNT),
-            "AWS_ECR_REPO_X86": codebuild.BuildEnvironmentVariable(value=LINUX_X86_ECR_REPO),
-            "AWS_ECR_REPO_AARCH": codebuild.BuildEnvironmentVariable(value=LINUX_AARCH_ECR_REPO),
-            "GITHUB_REPO_OWNER": codebuild.BuildEnvironmentVariable(value=GITHUB_REPO_OWNER),
+            "AWS_ACCOUNT_ID": codebuild.BuildEnvironmentVariable(value=env.account),
+            "AWS_ECR_REPO_X86": codebuild.BuildEnvironmentVariable(
+                value=LINUX_X86_ECR_REPO
+            ),
+            "AWS_ECR_REPO_AARCH": codebuild.BuildEnvironmentVariable(
+                value=LINUX_AARCH_ECR_REPO
+            ),
+            "GITHUB_REPO_OWNER": codebuild.BuildEnvironmentVariable(
+                value=GITHUB_REPO_OWNER
```

Comment:
`github_repo_name`?

Reply:
These variables are already there before this PR. But it looks like `GITHUB_REPO_OWNER` is no longer needed for Linux CodeBuild and can be safely removed